integration-tests/tests/testdata/correlation/stf-1.5.json

```diff
@@ -17624,7 +17624,7 @@
       "homepage": "NOASSERTION",
       "licenseComments": "Licensing information is automatically generated and may be incomplete or incorrect.",
       "licenseConcluded": "NOASSERTION",
-      "licenseDeclared": "GPLV2+,-LGPLV2+,-MIT",
+      "licenseDeclared": "NOASSERTION",
```

Comment:
So is this the fix? If so, I'd expect the test to fail consistently, but most of the time, it passes. Why?

Reply:
It always fails for me on a clean environment where I haven't ingested anything. If you have an old index with this document, or if you've run the walkers which also ingest this SBOM, then the test would pass. This explains why it doesn't fail when the tests run in integration tests staging.